app/graphql/resolvers/business/query.js

```diff
@@ -22,3 +23,13 @@ export const Query = {
     return transformOrganisationCSApplicationToBusinessApplications(response.applications)
   }
 }
+
+export const CPH = async ({ id }, _, { dataSources }) => {
```

Comment:
Could you move these into a separate file cph.js?
That would match the pattern we have elsewhere

Reply:
Absolutely, please see commit: 3906058